src/DocBlock/Annotation.php

```diff
@@ -157,7 +157,7 @@ public function getTypes()
      */
     public function setTypes(array $types)
     {
-        $pattern = '/'.preg_quote($this->getTypesContent()).'/';
+        $pattern = '/'.preg_quote($this->getTypesContent(), '/').'/';
```

Comment:
I'm slightly -0 here, no big deal.
just wanted to have solid reasoning behind that change (eg bugfix of some edge case we were not aware here, but as you checked - it's not a case here, thanks)